### ISSUE TITLE
Install failed 

### ISSUE THREAD

**Issue by ochees** [davidgohel/ReporteRs#127]
Hello,

I try to install this package (rJava works well) and i got an error, loading the package :
`Error in inDL(x, as.logical(local), as.logical(now), ...) : 
  unable to load shared object 'C:/Program Files/R/R-3.2.5/library/gdtools/libs/i386/gdtools.dll':
  LoadLibrary failure:  La procédure spécifiée est introuvable.
`

The sessioInfo :

```
R version 3.2.5 (2016-04-14)
Platform: i386-w64-mingw32/i386 (32-bit)
Running under: Windows XP (build 2600) Service Pack 3

locale:
[1] LC_COLLATE=French_France.1252  LC_CTYPE=French_France.1252    LC_MONETARY=French_France.1252 LC_NUMERIC=C                   LC_TIME=French_France.1252    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] ReporteRsjars_0.0.2 rJava_0.9-8        

loaded via a namespace (and not attached):
 [1] Rcpp_0.12.5       digest_0.6.9      withr_1.0.1       R.methodsS3_1.7.1 R6_2.1.2          httr_1.2.0        curl_0.9.7        R.oo_1.20.0       R.utils_2.3.0     xml2_0.1.2        devtools_1.11.1   tools_3.2.5      
[13] memoise_1.0.0

```

**Comment by davidgohel** [comment]
Hello,

It's says gdtools dll can not be found.
 
What if running:
```
install.packages("gdtools")
```

**Comment by ochees** [comment]
Hello,

Gdtools was already installed. I re-install `gdtools` in 3 ways : 

1) via `install.packages("gdtools")`. 
```
Installing package into ‘C:/Documents and Settings/xx/Mes documents/R/win-library/3.2’
(as ‘lib’ is unspecified)
--- Please select a CRAN mirror for use in this session ---
Warning: unable to access index for repository https://cloud.r-project.org/src/contrib:
  cannot open URL 'https://cloud.r-project.org/src/contrib/PACKAGES'
Warning: unable to access index for repository https://cloud.r-project.org/bin/windows/contrib/3.2:
  cannot open URL 'https://cloud.r-project.org/bin/windows/contrib/3.2/PACKAGES'
Warning message:
package ‘gdtools’ is not available (for R version 3.2.5)
```

2) via `install_github` , The install failed :
```
devtools::install_github("davidgohel/gdtools")
Downloading GitHub repo davidgohel/gdtools@master
from URL https://api.github.com/repos/davidgohel/gdtools/zipball/master
Installing gdtools
Skipping 1 unavailable package: Rcpp
"C:/PROGRA~1/R/R-32~1.5/bin/i386/R" --no-site-file --no-environ --no-save --no-restore --quiet CMD INSTALL "C:/Documents and Settings/xx/Local Settings/Temp/RtmpqUM2NX/devtools9ac6ee574e/davidgohel-gdtools-39cb542"  \
  --library="C:/Documents and Settings/xx/Mes documents/R/win-library/3.2" --install-tests 

* installing *source* package 'gdtools' ...
** libs
"C:/PROGRA~1/R/R-32~1.5/bin/i386/xxcript.exe" "../tools/winlibs.R"
Error in download.file("https://github.com/rwinlib/cairo/archive/v1.14.2.zip",  : 
  cannot open URL 'https://github.com/rwinlib/cairo/archive/v1.14.2.zip'
In addition: Warning message:
In download.file("https://github.com/rwinlib/cairo/archive/v1.14.2.zip",  :
  InternetOpenUrl failed: 'An error occurred in the secure channel support'
Execution halted
make: *** [winlibs] Error 1
Warning: running command 'make -f "Makevaxx.win" -f "C:/PROGRA~1/R/R-32~1.5/etc/i386/Makeconf" -f "C:/PROGRA~1/R/R-32~1.5/share/make/winshlib.mk" SHLIB_LDFLAGS='$(SHLIB_CXXLDFLAGS)' SHLIB_LD='$(SHLIB_CXXLD)' SHLIB="gdtools.dll" OBJECTS="CairoContext.o RcppExports.o font_metrics.o gdtools.o raster_to_base64.o"' had status 2
ERROR: compilation failed for package 'gdtools'
* removing 'C:/Documents and Settings/xx/Mes documents/R/win-library/3.2/gdtools'
* restoring previous 'C:/Documents and Settings/xx/Mes documents/R/win-library/3.2/gdtools'
Error: Command failed (1)
```
3) Install from zip files : appears as succesful
```
utils:::menuInstallLocal()
package ‘gdtools’ successfully unpacked and MD5 sums checked
```

In a second step, i load package ReporteRs -> ok

```
library(ReporteRs)
Loading required package: ReporteRsjars
```
Thanks for your help !

**Comment by davidgohel** [comment]
:) cool
